.gitignore

```diff
@@ -6,3 +6,4 @@ stories
 __pycache__
 
 .DS_Store
+dynamic/generators/booksinshelves/booksinshelves2.py
```

Comment:
Why was this added to the gitignore?

Reply:
Thank you for pointing this out! I will make that the habit. I just updated it accordingly.